Bindings/bindings.json

```diff
@@ -1474,6 +1474,14 @@
                     ]
                 },
                 {
+                    "name": "apiKey",
+                    "value": "string",
+                    "defaultValue": "SendGridApiKey",
```

Comment:
Yes, existing app settings will continue to work -- if we don't see an attribute, it'll flow up to the Environment variable that they already have. You're right they'll need to go through some pain to get to the new SendGrid model, though.